config.go

```diff
@@ -27,6 +27,7 @@ type Config struct {
 	PollInterval         string    `json:"poll-interval"`
 	PollIntervalDuration time.Duration
 	MaxRequestsPerQuery  int64 `json:"max-requests-per-query"`
+	MaxResponseSize      int64 `json:"max-client-response-size"`
```

Comment:
I don't think the naming is quite right here, you're configuring the max client response size of any internal clients created, not a limit on the response size bramble could return?